modules/heat_conduction/test/tests/radiation_transfer_action/tests

```diff
@@ -1,44 +1,64 @@
 [Tests]
```

Comment:
I'm a little confused by all the exodus files added in this gold's folder. Are they all used for tests?

Reply:
The exodus files in `radiation_transfer_action/tests/gold` are used; just checked that. Maybe I am misunderstanding the question here, though.